homeassistant/components/matter/light.py

```diff
@@ -376,6 +395,26 @@ def _update_from_device(self) -> None:
         else:
             self._attr_color_mode = ColorMode.ONOFF
 
+    def _check_transition_blocklist(self) -> None:
+        """Check if this device is reported to have non working transitions."""
+        device_info = self._endpoint.device_info
+        if (
+            device_info.vendorID,
+            device_info.productID,
+            device_info.hardwareVersionString,
+            device_info.softwareVersionString,
+        ) in TRANSITION_BLOCKLIST:
+            self._transitions_disabled = True
+        LOGGER.warning(
+            "Detected a device that has been reported to have firmware issues "
+            "with light transitions. Transitions will be disabled for this light"
+        )
```

Comment:
One level of indentation seems to be missing here.
```suggestion
            LOGGER.warning(
                "Detected a device that has been reported to have firmware issues "
                "with light transitions. Transitions will be disabled for this light"
            )
```

Reply:
oops clicked the commit button too fast, cleaned up the mess